roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -90,6 +90,22 @@
       when:
         - rpm_query_result.rc > 0
 
+- name: Bootstrap command
```

Comment:
If we assume this is used to run some commands like subscription-manager then this should be executed before L76, because that task installs the rhosp-release package

Reply:
Indeed! Not sure how I missed that. Thanks, i'll move it earlier.